app/scripts/background.js

```diff
@@ -1309,7 +1301,9 @@ async function onInstall() {
   // and is therefore an install
   if (process.env.IN_TEST) {
     addAppInstalledEvent();
-  } else if (!storeAlreadyExisted && !process.env.METAMASK_DEBUG) {
+  } else if (storeAlreadyExisted && !process.env.METAMASK_DEBUG) {
```

Comment:
What is the reason for this change? It looks like we're now treating all pre-existing installations as "first time installs", which definitely isn't right.

Where is the `isFirstTimeInstall` variable referenced in the comment?

Reply:
both of those were mistakes, fixed here e4f03bc4a7